admin/banner_manager.php

```diff
@@ -130,7 +130,7 @@
 
           $sql = "UPDATE " . TABLE_BANNERS . "
                   SET
-                    date_scheduled = :scheduledDate,
+                    date_scheduled = DATE_ADD(:scheduledDate, INTERVAL '00:00:00' HOUR_SECOND),

```

Comment:
Then the `if` statements around those already-initialized vars on lines 137-145 are not needed, and should be removed, leaving just the `$sql=$db->bindVars()` calls because the `if`s have no purpose, because they'll always be true.